pkg-manager/resolve-dependencies/src/resolveDependencies.ts

```diff
@@ -1252,15 +1250,15 @@ async function resolveDependency (
         : options.parentPkg.rootDir,
       registry: wantedDependency.alias && pickRegistryForPackage(ctx.registries, wantedDependency.alias, wantedDependency.pref) || ctx.registries.default,
       skipFetch: ctx.dryRun,
-      update: options.update,
+      update: options.update === 'in-range',
+      updateToLatest: options.update === 'latest',
```

Comment:
why don't we pass down the same option?

Reply:
Yay, I was initially hesitant to make further changes, but I agree that it would be more consistent. I made the changes to the package requesting now in a8a68252cd0d67d0a5f2c56a689ea096840e2a02